docs/src/tutorial/cython_tutorial.rst

```diff
@@ -238,12 +301,20 @@ which we can try out in the interactive interpreter as follows::
 See, it works! And if you're curious about how much work Cython has saved you,
 take a look at the C code generated for this module.
 
-
 Cython has a way to visualise where interaction with Python objects and
 Python's C-API is taking place. For this, pass the
 ``annotate=True`` parameter to ``cythonize()``. It produces a HTML file. Let's see:
 
-.. figure:: htmlreport.png
+.. tabs::
+    .. group-tab:: Cython
+
+        .. figure:: htmlreport.png
+            :scale: 90 %
+
+    .. group-tab:: Pure Python
+
+        .. figure:: htmlreport_pure.png
+            :scale: 90 %
```

Comment:
I really don't like the `_pure` suffix. We should avoid it if possible and use balanced names instead (where they make sense).
```suggestion
.. tabs::
    .. group-tab:: Cython

        .. figure:: htmlreport_pyx.png
            :scale: 90 %

    .. group-tab:: Pure Python

        .. figure:: htmlreport_py.png
            :scale: 90 %
```

Reply:
I accept the suggestion, at the same time changing [this](https://github.com/cython/cython/pull/4226/commits/b862b36044d95abdfbad2f4b95efc1d833f0dcd4) and [that](https://github.com/cython/cython/pull/4226/commits/5febe377a28cb574ce9b60b96d7331388cfc685d)